util/fipstools/acvp/modulewrapper/modulewrapper.cc

```diff
@@ -268,18 +268,28 @@ static bool GetConfig(const Span<const uint8_t> args[], ReplyCallback write_repl
       },
       {
         "algorithm": "SHAKE-128",
-        "revision": "2.0",
+        "revision": "1.0",
```

Comment:
Why is this going back to an older version? Is 1 the right thing to be testing?

Reply:
Revision 2.0 doesn't exist as of right now: https://pages.nist.gov/ACVP/draft-celi-acvp-sha3.html#name-sha3-and-shake-algorithm-ca

Reply:
Did it used to exist? I'm confused what happened here.